modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
@baartosz want to test&adopt your httplug client + guzzle. Good time to make it a try? would I wait a little for your " I'll update this PR soon with default version.."

thx a lot for your work